package.json

```diff
@@ -28,6 +28,9 @@
   "dependencies": {
     "matched": "^0.4.4"
   },
+  "peerDependencies": {
+    "rollup": "^0.48.0"
+  },
```

Comment:
I'm not sure about this change. Before, the rollup version was only mentioned in the README, now npm also knows about it, and will generate warnings for users who have an incompatible version. The downside is that rollup tends to release new major versions quite rapidly, and every time they do, this field will have to be updated to avoid the warnings.